xbmc/platform/darwin/osx/SDLMain.mm

```diff
@@ -556,7 +556,8 @@ int main(int argc, char *argv[])
     status = SDL_main(gArgc, gArgv);
     SDL_Quit();
 
-    [xbmc_delegate applicationWillTerminate:NULL];
```

Comment:
so, why change this? The warning didn't appear because of deployment target raise, it had been there before.

Reply:
I figured one less warning was better 😉

Reply:
well, creating a fake notification just to silence warning isn't a proper fix :) You'd rather create a new method that will be called by this code and by `-applicationWillTerminate:`. But this really looks unrelated to the PR.

Reply:
Fair, will remove